processor/src/advice/inputs.rs

```diff
@@ -10,54 +10,54 @@ use super::{utils::IntoBytes, BTreeMap, Felt, InputError, MerkleSet, Vec};
 ///
 /// There are three types of advice inputs:
 ///
-/// 1. Single advice tape which can contain any number of elements.
-/// 2. Multiple advice tapes that can be appended to the main tape, and are mapped by 32 bytes keys.
+/// 1. Single advice stack which can contain any number of elements.
+/// 2. Multiple advice stacks that can be appended to the main stack, and are mapped by 32 bytes keys.
```

Comment:
We should update the second point here.

Reply:
You mean the term "main stack"? If yes, done!